src/test/java/SetTest.java

```diff
@@ -0,0 +1,48 @@
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class SetTest {
+
+    private Set<Integer> numbers;
+
+    @BeforeEach
+    void setUp() {
+        numbers = new HashSet<>();
+        numbers.add(1);
+        numbers.add(1);
+        numbers.add(2);
+        numbers.add(3);
+    }
+
+    @DisplayName("size() 메소드를 활용해 Set의 크기를 확인하는 학습테스트 구현")
+    @Test
+    void set_메소드_학습테스트() {
+        int actual = numbers.size();
+
+        assertThat(actual).isEqualTo(3);
+    }
+
+    @DisplayName("contains() 메소드를 활용해 1, 2, 3의 값이 존재하는지를 확인하는 학습테스트 구현")
+    @ParameterizedTest
+    @ValueSource(ints = {1,2,3})
+    void contains_메소드_학습테스트(int actual) {
+        assertThat(numbers.contains(actual)).isTrue();
```

Comment:
```suggestion
        assertThat(numbers).contains(actual);
```

이렇게도 가능합니다. 그런데, 매개변수 값은 실제 값이라기보단 기대값이기 때문에 actual보다는 expected가 좀 더 적절해보이네요!